src/api/replication_api.js

```diff
@@ -14,11 +14,11 @@ module.exports = {
 
     methods: {
 
-        move_objects_by_type: {
+        copy_objects_by_type: {
```

Comment:
If the copy_type is not a requirement anymore, I don’t see a reason to preserve the `_by_type` suffix.
```suggestion
        copy_objects: {
```

Reply:
@dannyzaken Can you pls confirm on this, I am not sure of it.

Reply:
I don't have a strong opinion on that. Either way will work

Reply:
Okay, I think for now we can keep it as same if in future we will be implementing other types of server side copy. Otherwise we can remove it @liranmauda wdyt its a very small change?

Reply:
I don't think we should change the api itself, just it's name.

Reply:
@liranmauda Done, PTAL.